platform/src/ConsolePanel.js

```diff
@@ -13,11 +14,15 @@ class ConsolePanel extends Panel {
     }
 
     getButtons() {
```

Comment:
Where is this method used?

Reply:
It's called in the ConsolePanel constructor on line 11 which sets the member variable that gets used to render the buttons.

Reply:
Panel defines an `addButtons` method. Why is that not used in the `ConsolePanel` constructor?